Don't show img element if user doesn't have a profile picture

SEQUOIIA opened the pull request:
Issue: https://github.com/dfds/cloudplatform/issues/1906

Before:
![Screenshot 2023-08-17 at 1 41 59@2x](https://github.com/dfds/selfservice-portal/assets/1633308/9a3c0d36-1bea-4ce6-aa63-5f886f348fcd)
After:
![Screenshot 2023-08-17 at 1 42 07@2x](https://github.com/dfds/selfservice-portal/assets/1633308/4f85d0c6-116b-44ca-899c-f7e8338f7289)

kralle333 reviewed (changes requested):
Nice!

Can you instead show an image like this?
![image](https://github.com/dfds/selfservice-portal/assets/5738476/1f32b961-9aa3-44c0-a6d4-5c9b830c0eaf)

think its pretty standard user has no image set thing

SEQUOIIA commented:
> Nice!
> 
> Can you instead show an image like this? ![image](https://user-images.githubusercontent.com/5738476/261587190-1f32b961-9aa3-44c0-a6d4-5c9b830c0eaf.png)
> 
> think its pretty standard user has no image set thing

Fixed in https://github.com/dfds/selfservice-portal/pull/45/commits/8e82df35f8a34b3708ed1f5c98aa88e653f79724.

I also noticed that the centering of members were a bit off if a member's name is too long. I fixed that in https://github.com/dfds/selfservice-portal/pull/45/commits/c2d7fd1a41f8d5b2967f34186a5446539a0983d7

Here's a before & after of that:

Before:
![Screenshot 2023-08-18 at 2 27 00@2x](https://github.com/dfds/selfservice-portal/assets/1633308/aeeb3d6d-06cb-4b2b-b18e-4a3d79c80d29)
After:
![Screenshot 2023-08-18 at 2 29 13@2x](https://github.com/dfds/selfservice-portal/assets/1633308/51666a67-888c-4c34-b819-488e963fc8c2)

kralle333 reviewed (approved):
😍
